### ISSUE TITLE
Invalid hook call when trying to use in React 17.0.2

### ISSUE THREAD

**Issue by pkawula** [NdYAG/react-rater#64]
I wanted to use this plugin but when I add it to my component it crashes whole app
```
Error
Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
```

Here's the reproduced bug
(Codesandbox)[https://codesandbox.io/s/react-rater-invalid-hook-call-kb5mp3?file=/src/App.js]

**Comment by NdYAG** [comment]
@pkawula Sorry about late update. This module is updated to support React up to 18 now. Please install `6.0.1`, if there's any question please let me know, thanks

**Comment by SaadBazaz** [comment]
Still getting this error when using React 17 and NextJS.

In the console:
```
Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
```

On the webpage:
```
Unhandled Runtime Error
TypeError: Cannot read properties of null (reading 'useState')
```

Versions:
    "next": "11.1.0",
    "react": "17.0.2", 
    "react-rater": "^6.0.1",

**Comment by AZKZero** [comment]
+1 Having the same issue

**Comment by NdYAG** [comment]
Sorry for the late reply. This issue is likely caused by React version mismatch as in 6.0.1 react-rater ask to have react version as ^18.0.0, this is been corrected now in 6.0.2. For anyone who have the same issue please use react-rater 6.0.2, it should work with your react version.

**Comment by SaadBazaz** [comment]
Still not working with ```"react": "^17.0.2"```.
